resources/js/components/journal/partials/JournalContentEntry.vue

```diff
@@ -30,6 +30,11 @@
             <h3 class="mb1">
               {{ entry.title }}
             </h3>
+            <p class="mb1">
+              <span class="pr2 f6 avenir">
+                {{ $t('journal.journal_created_at', { date: entry.created_at }) }}
+              </span>
+            </p>
```

Comment:
I have the feeling this will add too much information on the page.
Try changing line 11 with this instead:
```
      <div class="br3 ba b--gray-monica bg-white pr3 pb3 pt3 mb3 journal-line"
        v-tooltip.top="$t('journal.journal_created_at', { date: entry.created_at })"
      >
```

Reply:
Done!